packages/web/src/scss/components/Breadcrumbs/README.md

```diff
@@ -0,0 +1,25 @@
+# Breadcrumbs
+
+Breadcrumbs present a trail to each previous page in relation to website's structure.
```

Comment:
Just FYI: once #424 and #425 are merged, we'll start keeping those common descriptions in Supernova because they are valid for components as such, not just for the particular technology. I've added it [there](https://spirit.supernova-docs.io/spirit/latest/components/breadcrumbs/html-1.html) so you can remove it from here.

Reply:
@adamkudrna you mean the whole file or just this line? Is this text about to become some kind of Supernova placeholder between heading and code example?

Reply:
Just this line can be removed. I moved it to Supernova:

![obrazek](https://user-images.githubusercontent.com/5614085/185417530-dcf3770c-8f89-4e2c-bbf5-7aff85c10e7b.png)